core/src/avm2/object/array_object.rs

```diff
@@ -217,13 +208,12 @@ impl<'gc> TObject<'gc> for ArrayObject<'gc> {
         let array_length = read.array.length() as u32;
 
         // Array enumeration skips over holes.
-        while last_index < array_length {
-            if read.array.get(last_index as usize).is_some() {
-                return Ok(Some(last_index + 1));
-            }
-            last_index += 1;
+        if let Some(index) = read.array.get_next_enumerant(last_index as usize) {
+            return Ok(Some(index as u32));
         }
 
+        last_index = max(last_index, array_length);
```

Comment:
Can this just use `std::cmp::max` here instead of doing a
```
use std::cmp::max;
```
at the top of the code?